package.json

```diff
@@ -5,19 +5,20 @@
     "type": "git",
     "url": "https://github.com/mobify/split-test"
   },
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
Although I feel like maybe changing it to 0.2.0 would make sense, because then it's less likely someone will run a `bower install` in the future and have something change from underneath them! (because most of our stuff matches ~)